.travis.yml

```diff
@@ -18,6 +21,15 @@ before_install:
 
 install:
   - dep ensure
+  - docker build -t eduboard/backend .
```

Comment:
Since here the app is already running on Travis locally anyways, does it make sense to actually build an image instead of just launching MongoDB from Docker using docker-compose?

Reply:
I wanted to test the Dockerfile as well and i guess a pretty basic test where we look if the container can start helps there a bit.
It could be replaced with more refurbished tests though if we want.

Reply:
I doubt the Dockerfile will change often so I would not test that. If you still want to go that way though, we could instead test inside Docker exclusively.
To be honest I don't think there is much benefit to that though, since building that image takes quite some time usually.